src/test/java/uk/gov/justice/laa/crime/cfecrime/utils/InitMeansTestOutcomeCalculatorTest.java

```diff
@@ -0,0 +1,87 @@
+package uk.gov.justice.laa.crime.cfecrime.utils;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.justice.laa.crime.cfecrime.Exceptions.UndefinedOutcomeException;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.MeansTestOutcome;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.InitAssessmentResult;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class InitMeansTestOutcomeCalculatorTest {
+
+    private MeansTestOutcome meansTestOutcome;
+
+    private boolean fullAssessmentPossible;
+    private InitAssessmentResult initAssessmentResult;
+
+    @Test
+    public void FAILInitResultAndAssementNotPossible() throws UndefinedOutcomeException {
+        initAssessmentResult = InitAssessmentResult.FAIL;
+        fullAssessmentPossible = false;
+
+        MeansTestOutcome oc = InitMeansTestOutcomeCalculator.getInitMeansTestOutcome(initAssessmentResult,fullAssessmentPossible);
+
+        assertEquals(oc, MeansTestOutcome.INELIGIBLE);
+    }
+
+    @Test
+    public void FAILInitResultAndAssementPossible() throws UndefinedOutcomeException {
+        initAssessmentResult = InitAssessmentResult.FAIL;
+        fullAssessmentPossible = true;
+
+        MeansTestOutcome oc = InitMeansTestOutcomeCalculator.getInitMeansTestOutcome(initAssessmentResult,fullAssessmentPossible);
+
+        assertEquals(oc, null);
+    }
+
+    @Test
+    public void PASSInitResultAndAssementPossible() throws UndefinedOutcomeException {
+        initAssessmentResult = InitAssessmentResult.PASS;
+        fullAssessmentPossible = true;
```

Comment:
This combo isn't possible - PASS always returns false. 

[This feels like we should have some sort of 'protection' layer between the controller and CMA so that the outputs received always make sense - but maybe after we tackle the CMA bit]